test/tests/functional/pbs_equiv_classes.py

```diff
@@ -48,6 +48,12 @@ class TestEquivClass(TestFunctional):
     """
 
     def setUp(self):
+        # Skip the test on cgroup cpuset system
```

Comment:
Rather than put these test for cgroup cpuset system here, I suggest moving them to a function in class TestFunctional (PBSTestsuite) (test/fw/lib/pbs_testsuite.py). So then it can be called later from other testsuites, not this one. If it's not too much trouble, perhaps merge the common codes in class MoM.is_cpuset_mom() with this this TestFunctional.is_cpuset(hostname>...